checkpoint/cp-resource-management/templates/_helpers.tpl

```diff
@@ -25,4 +25,18 @@ If release name contains chart name it will be used as a full name.
 {{- else -}}
     {{ default "default" .Values.serviceAccount.name }}
 {{- end -}}
-{{- end -}}
\ No newline at end of file
+{{- end -}}
+
+{{- define "dome9.url" -}}
+{{- $region := default "us1" .Values.region -}}
+{{- if eq $region "us1" -}}
+{{- printf "https://api-cpx.dome9.com" -}}
+{{- else if eq $region "eu1" -}}
+{{- printf "https://api-cpx.eu1.dome9.com" -}}
+{{- else if eq $region "ap1" -}}
+{{- printf "https://api-cpx.ap1.dome9.com" -}}
+{{- else -}}
+{{- $err := printf "\n\nERROR: Invalid region -- %s" .Values.region -}}

```

Comment:
I would also add:
The supported regions are: 'us1' (N. Virginia), 'eu1' (Ireland), 'ap1' (Singapore)

(we should check if it's really N. Virginia and Ireland)

Reply:
the regions names are correct..
Do we prefer hardcoded URLs and not "api-cpx." + region + ".dome9.com" ? Unless we want to verify regions - then we can use [has/mustHas](http://masterminds.github.io/sprig/lists.html)? With US as a special case?

Reply:
since there is special case, and we have only 3 regions - I think having it like this is easier to understand. When we have more regions (e.g., 6) - would make sense to refactor.

Reply:
@chkp-yonatanph regarding listing supported regions - IMO, better place for it is the readme (and duplicating the same info in several places makes it more likely that we will forget one of them in the future).

Reply:
This is a VERY typical error in many cli tools (amazon, gcp, etc.).
Personally, when the CLI output gives me the correct option it's MUCH nicer than sending me to search for the right option in some document.
From a usability standpoint, cli suggestion/correction is the right approach.

Going forward we will probably want to add a second special case for 'free-text'.
Example: 
--set region=other --set url=https://<URL>
-->
{{- if eq $region "other" -}}
{{- printf url -}}

Reply:
OK. Then we should have a list/array (as Igor suggested) - we can use it both to print the error and do validation.
Regarding 'set url' - it can just override the region, without the need for 'other'.

Reply:
in my proposal (with has/mushHas) we will need to have a list of valid regions.
In Readme I would point to values file and SK with list of supported regions for each feature
@chkp-alexgl I think it worth updating values.yaml with `region`